index.js

```diff
@@ -75,8 +75,6 @@ export default function ({
 				})};\n`
 			);
 
-			builder.copy(join(files, 'api'), apiDir);
```

Comment:
I think we still need this line - without it, we no longer copy over `/files/api/function.json` to `api/render`.

Reply:
You're right. I got mixed up about which folder that was copying.